### ISSUE TITLE
net core2.1 datatable select Occasionally you don't get the right value

### ISSUE THREAD

**Issue by dengyantao** [dotnet/aspnetcore#19757]
net core2.1 datatable select Occasionally you don't get the right value
example：
DataTable dttaxpayable =new DataTable ();
dttaxpayable .Select("idnumber = '" + item.Idnumber + "' ")

**Comment by javiercn** [comment]
@dengyantao thanks for contacting us.

We don't believe this issue is about ASP.NET Core. In order for you to get a better and faster answer, I would recommend you file the issue in the [dotnet/runtime](https://github.com/dotnet/runtime) repository.

**Comment by javiercn** [comment]
@ajcvickers is this EF by any chance?

**Comment by ajcvickers** [comment]
@javiercn No. DataTable is not EF. https://github.com/dotnet/runtime is the correct place.
